johnsonandjohnson/Bodiless-JS
# fix(filtering): Fix FilterByGroup body accordion hidden on load

## Changes

Fix an issue where FilterByGroup filters appear in an aria hidden accordion by default on desktop, which cause accessibility issues. This issue only happens on production, since it's caused by SSR. 

Note that this change doesn't affect how filter categories are rendered on mobile.

## Test Instructions

- Run the development server with `npm run dev`
- Go to `/filter-page/`
- Enter edit mode and create some categories
- Quit the development server and create a new build with `npm run build`
- Run a production server with `npm run serve`
- Open `/filter-page/` again using a large browser window (at least 1024px wide)

Before the change:
- The categories are hidden and inaccessible via the keyboard

After the change
- The categories appear normally and are accessible using a keyboard
